src/templates/ec2/minio.yaml

```diff
@@ -0,0 +1,201 @@
+          rules:
+            - host: '{{ $.cndi.prompts.responses.argocdDomainName }}'
+              http:
+                paths:
+                  - path: /
+                    pathType: Prefix
+                    backend:
+                      service:
+                        name: argocd-server
+                        port:
+                          name: https
+      tenant-server-ingress:
+        apiVersion: networking.k8s.io/v1
+        kind: Ingress
+        metadata:
+          name: tenant-server-ingress
+          namespace: '{{ $.cndi.prompts.responses.tenantNamespace }}'
+          annotations:
+            cert-manager.io/cluster-issuer: cluster-issuer
+            kubernetes.io/tls-acme: 'true'
+            nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+            nginx.ingress.kubernetes.io/backend-protocol: HTTPS
+        spec:
+          tls:
+          - hosts:
+            - '{{ $.cndi.prompts.responses.tenantDomainName }}'
+            secretName: cluster-issuer-private-key
+          rules:
+          - host: '{{ $.cndi.prompts.responses.tenantDomainName }}'
+            http:
+              paths:
+              - path: "/"
+                pathType: Prefix
+                backend:
+                  service:
+                    name: minio
+                    port:
+                      name: https-minio
+      myminio-tenant-config:
+        apiVersion: v1
+        kind: Secret
+        metadata:
+          name: myminio-tenant-config
+          namespace: '{{ $.cndi.prompts.responses.tenantNamespace }}'
+          annotations:
+            sealedsecrets.bitnami.com/cluster-wide: "true"
+          labels:
+            argocd.argoproj.io/secret-type: cluster
+            app.kubernetes.io/instance: tenant
+        type: Opaque
+        stringData:
+          config.env: $.cndi.secrets.seal(MINIO_TENANT_CREDENTIALS)
+      tenant-crd:
+          apiVersion: minio.min.io/v2
+          kind: Tenant
+          metadata:
+            annotations:
+              argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+              argocd.argoproj.io/hook: Sync
+            name: '{{ $.cndi.prompts.responses.tenantName }}'
+            namespace: '{{ $.cndi.prompts.responses.tenantNamespace }}'
+            labels:
+              app: minio
+          spec:
+            configuration:
+              name: myminio-tenant-config
+            pools:
+              - servers: '{{ $.cndi.prompts.responses.tenantServers }}'
+                volumesPerServer: '{{ $.cndi.prompts.responses.VolumesPerServer }}'
+                volumeClaimTemplate:
+                  metadata:
+                    name: data
+                  spec:
+                    accessModes:
+                      - ReadWriteOnce
+                    resources:
+                      requests:
+                        storage: 10Gi
+            buckets:
+              - name: '{{ $.cndi.prompts.responses.VolumesPerServer }}'
```

Comment:
this is meant to be the name of the bucket, but instead returns an integer

Reply:
Fixed that.